### PR TITLE
AB#300 Add accordion styles

### PR DESCRIPTION
**Pull request by helloimela**
- Added styling to accordion
- Add it as web component
- It is also available as classes including expanded state

- Check Figma > UI Kit Global > Accordion
 FIX for [AB#300](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/300)

### CODE REVIEW

**Comment by niklasros** [comment]
Possible to add the arrow-animation as we have currently in the SDDS left navigation?

**Comment by helloimela** [comment]
> Possible to add the arrow-animation as we have currently in the SDDS left navigation?

@niklasros yeah absolutely!

**Review by niklasros — APPROVED** [review]
Looks good!

File: components/src/components/accordion/accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+} from '@stencil/core';
+
+@Component({
+  tag: 'sdds-accordion',
+  styleUrl: 'accordion.scss',
+  shadow: true,
+})
+
+export class Accordion {
+  @Element() el: HTMLElement;
```
Comment (mmexvr) [inline components/src/components/accordion/accordion.tsx:12]: Small clean up

File: components/src/components/accordion/accordion.stories.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import readme from './readme.md';
+
+export default {
+  title: 'Component/Accordion',
+  argTypes: {
+    affix: {
+      defaultValue: 'suffix',
```
Comment (mmexvr) [inline components/src/components/accordion/accordion.stories.js:7]: Nothing that needs attention now, for some reason(not sure why) default value doesn't show in the control table and adds a "-" in the column of the description. Looks a bit odd 🤔 

Other way is just adding args: { affix: 'suffix' } in the export default {}

**Review by mmexvr — APPROVED** [review]

**Pull request merged**